metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -68,6 +61,9 @@ class KafkaSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigBase):
     topic_subject_map: Dict[str, str] = dict()
     # Custom Stateful Ingestion settings
     stateful_ingestion: Optional[KafkaSourceStatefulIngestionConfig] = None
+    schema_registry_class: str = (
```

Comment:
Please update https://github.com/acryldata/datahub-fork/blob/master/metadata-ingestion/source_docs/kafka.md with the documentation of this new config param.